docs/reference/search-application/apis/search-application-search.asciidoc

```diff
@@ -57,21 +70,16 @@ PUT _application/search_application/my-app
         "query": {
           "multi_match": {
             "query": "{{query_string}}",
-            "fields": [{{#text_fields}}"{{name}}^{{boost}}"{{^last}},{{/last}}{{/text_fields}}]
+            "fields": ["title^{{title_boost}}", "description"]
```

Comment:
Same comment here on fields. I don't think we should omit this, but I'm fine adding a simpler example too.